features.md

```diff
@@ -41,12 +41,50 @@
    
 * ### Working Offline
     <!--Explain a bigger picture of sync on/off feature and how to switch-->
+    The openMRS android client can also be sucessfully used in areas with limited connectivity. As you open the OpenMRS android client    the login screen appears. Here you may provide your login credentials and a location (e.g. "Registration Desk")
```

Comment:
Would you mind capitalizing `openMRS android client` to `OpenMRS Android Client`? Please also remove the extra space preceding "the login screen".